opt_out_http_api/api_methods.py

```diff
@@ -19,46 +18,26 @@ class OptOutNotDeleted(Exception):
 class API(object):
     app = Klein()
 
-    def __init__(self):
-        self._optouts = [
-            {"id": "2468", "address_type": "msisdn", "address": "+273121100"},
-            {"id": "5678", "address_type": "twitter",
-             "address": "@twitter_handle"}
-        ]
+    def __init__(self, backend_class):
+        self._backend = backend_class()
 
 # Get Opt Out Address
 
     def get_opt_out(self, addresstype, address):
-        opt_outs = [
-            o for o in self._optouts
-            if o["address_type"] == addresstype and o["address"] == address
-        ]
-        if opt_outs:
-            return opt_outs[0]
-        return None
+        return self._backend.get(addresstype, address)
```

Comment:
I would use `self._backend.get` directly rather than keeping the `get_opt_out` function (since it just calls the backend with the same parameters).